extmod/machine_i2s.c

```diff
@@ -600,6 +600,7 @@ STATIC mp_uint_t machine_i2s_stream_write(mp_obj_t self_in, const void *buf_in,
 
         return size;
     } else { // blocking or asyncio mode
+
```

Comment:
I think this blank line can be removed, to keep coding style consistent (eg with the "if" part of this statement).